lib/Thingiverse.pm

```diff
@@ -60,8 +52,19 @@ our $client_id          = 'c587f0f2ee04adbe719b';
 our $access_token       = 'b053a0798c50a84fbb80e66e51bba9c4';
```

Comment:
Concerning the access token.  It is only a temporary, private access code, according to the following excerpt from http://www.thingiverse.com/developers/getting-started (near the bottom):

Application Types
There are four kinds of applications that can be built with the Thingiverse API:
Web - An application that operates in a web browser, but outside the Thingiverse site.
Iframe - An application that operates within the Thingiverse site by loading inside of an iframe. These apps can use the Thingiverse Javascript API to present dialogs to the user for common functions.
Mobile - A standalone application that will operate in a mobile or tablet environment.
Desktop - A standalone application that will operate on a desktop computer environment.
Web and Iframe apps will need to specify the URL where the app lives and should include a callback URL.

Your app will be private until you publish it and the Thingiverse moderators review and approve it. While your application is private you can have up to 10 people register and play with it.